pkg/commands/process/settings/rules/javascript/third_parties/segment.yml

```diff
@@ -0,0 +1,64 @@
+patterns:
+  - pattern: |
+      $<INSTANCE>.$<METHOD>($<...>$<DATA_TYPE>$<...>)
+    filters:
+      - variable: INSTANCE
+        detection: javascript_third_parties_segment_instance
+      - variable: DATA_TYPE
+        detection: datatype
+      - variable: METHOD
+        values:
+          - alias
+          - identify
+          - group
+          - page
+          - track
+auxiliary:
+  - id: javascript_third_parties_segment_module
+    patterns:
+      - |
+        require('@segment/analytics-node')
```

Comment:
in javascript all of these are valid, I would try to avoid require as it is a bit outdated way to import
```js
const segment = require('segment')
import segment from 'segment'
// this one is a bit unique as it takes only a part of exported object
import { createClient } from 'segment'
```

Reply:
Agreed; I'll update it 🙏